src/components/pages/Vehicles.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable */
+import React, { useEffect, useState } from 'react'
+import './styles/vehicle.css'
+// import img from '../../assets/bike.jpg'
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom'
+import { getVehicles } from '../../redux/vehicles/vehicles';
+// Import Swiper React components
+import { Swiper, SwiperSlide } from 'swiper/react';
+// Import Swiper styles
+import 'swiper/css';
+// import { createAsyncThunk } from '@reduxjs/toolkit';
+// import VehiclesDetails from './VehiclesDetails';
+// import { getVehicleId } from '../../redux/vehicles/vehicles';
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import 'swiper/css/scrollbar';
+
+const Vehicles = () => {

```

Comment:
- [ ] There is functionality already in the backend to delete motorcycles, but I couldn't find any way to delete a motorcycle in the frontend implementation.

Since your team has 4 members, this is a required feature kindly implement it.

Tip:  Just add a "Delete motorcycle" link in the navigation panel that leads to a page with a list of the motorcycles.  Each item with a "delete" button.  

Since you already have the backend functionality, this should be trivial. 💪